README.md

```diff
@@ -1,2 +1,36 @@
 # wasmcloud-provider-couchbase
-Wasmcloud Couchbase capabilities provider
+
+This is a capability provider for wasmCloud to provide Couchbase KV connectivity to Wasm applications via `wasi-keyvalue`. At the moment it supports the `wasi:keyvalue/store@0.2.0-draft` interface.
+
+This provider uses the **RawJSONTranscoder** for Couchbase, storing any new keys as binary data. Since the wasi-keyvalue interface works entirely in storing and retrieving binary data, the deserialization into a `struct` or structured data must be done on the component side.
+
+## Build
+
+Prerequisites:
+
+- [wash 0.29](https://wasmcloud.com/docs/installation) or later
```

Comment:
Just a heads up this isn't technically released yet, but is very high on the priority to release!